build.cake

```diff
@@ -1,3 +1,5 @@
+#tool "nuget:?package=GitVersion.CommandLine&version=3.5.4"
+
```

Comment:
Yea, we unfortunately can't yet get the result from GitVersionTask available in the Cake script, so we need to run `gitversion.exe` as well. It will produce the same version number, though, so it's only a theoretical performance issue. I'd like to avoid it, but currently that is not possible.